pkg/client/block/godoc.go

```diff
@@ -0,0 +1,5 @@
+// Package block contains the client implementation of the EventsReplayClient
```

Comment:
This statement is no longer true as the `EventsReplayClient` is encapsulated now.

Reply:
WDYT of the update? @Olshansk @bryanchriswhite